pkg/generate/app/sourcelookup.go

```diff
@@ -260,7 +260,7 @@ func (r *SourceRepository) RemoteURL() (*url.URL, bool, error) {
 	case "file":
 		gitRepo := git.NewRepository()
 		remote, ok, err := gitRepo.GetOriginURL(r.url.Path)
-		if err != nil && err != git.ErrGitNotAvailable {
+		if err != git.ErrGitNotAvailable {
```

Comment:
This changes the behavior of the method.  With this change if you have a `nil` error and `ok == true`, you'll return here with an incorrect (`false`) return value.